tools/buildstamp/get_workspace_status

```diff
@@ -14,6 +14,17 @@ set -e
 # If the script exits with non-zero code, it's considered as a failure
 # and the output will be discarded.
 
+# Provide an escape hatch for people building from github release tarballs,
+# but in non-Sorbet git repositories, to get somewhat more accurate version
+# information.
+if [ -f .github/SORBET_ARCHIVE_SHA ]; then
+    git_rev=$(head -n 1 .github/SORBET_ARCHIVE_SHA)
+    echo "STABLE_BUILD_SCM_REVISION ${git_rev}"
+    echo "STABLE_BUILD_SCM_COMMIT_COUNT 0"
```

Comment:
This commit count gets used to compute the version number, like `0.5.10301`.

Do you think we could get that information as well?

Reply:
I don't think so: getting the commit count requires access to the git repository (or at least the git checkout), which we don't have.

The tarball available as a github release does contain the version number information in the unzipped directory, so you could parse things out of that directory name, but the tarball on the Stripe side for whatever reason only has the SHA.